.github/workflows/first-actions.yml

```
@@ -4,25 +4,33 @@ on: [push]
 
     strategy:
       matrix:
         python-version: [3.8, 3.9]
 
     steps:
-    - uses: actions/checkout@v3
-    - name: Set up Python
-      uses: actions/setup-python@v2
-      with:
-        python-version: ${{ matrix.python-version }}
+      - uses: actions/checkout@v3
```

actionlint (1.7.7)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)